### PR TITLE
Update Office ASP.NET SSO sample

### PR DESCRIPTION
**Pull request by davidchesnut**
|        Q        |                    A                    |
| --------------- | --------------------------------------- |
| Bug fix?        | no                                |
| New feature?    | no                                |
| New sample?     | no                               |
| Related issues? | --- |

## What's in this Pull Request?

- Remove fallback code as it now requires a second app registration which is a bit complex to include in this sample at this time.
- Upgrade to use ASP.NET Core.

### CODE REVIEW

**Review by AlexJerabek — COMMENTED** [review]
Was the readme for this sample updated? The folder names and the middle-tier warning seem out of date.

**Comment by davidchesnut** [comment]
@AlexJerabek I updated the readme, so should be good now. Thanks!

File: Samples/auth/Office-Add-in-ASPNET-SSO/README.md [hunk 1]
```diff
@@ -23,39 +23,34 @@ extensions:
...
-
 There are two versions of the sample in this repo, one of which has its own README file:
 
-- **Before** folder. The starting point for the SSO walkthrough at [Create an ASP.NET Office Add-in that uses single sign-on](https://docs.microsoft.com/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet). Please follow the instructions in the article.
-- **Complete** folder. The completed sample. To use this version, follow the instructions in the article [Create an ASP.NET Office Add-in that uses single sign-on](https://docs.microsoft.com/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet), but substitute "Complete" for "Before" in those instructions and skip the sections **Code the client-side** and **Code the server-side**.
+- **Begin** folder. The starting point for the SSO walkthrough at [Create an ASP.NET Office Add-in that uses single sign-on](https://docs.microsoft.com/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet). Please follow the instructions in the article.
```
Comment (AlexJerabek) [inline Samples/auth/Office-Add-in-ASPNET-SSO/README.md:28]:
```suggestion
- **Begin** folder. The starting point for the SSO walkthrough at [Create an ASP.NET Office Add-in that uses single sign-on](https://learn.microsoft.com/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet). Please follow the instructions in the article.
```

File: Samples/auth/Office-Add-in-ASPNET-SSO/README.md [hunk 2]
```diff
@@ -23,39 +23,34 @@ extensions:
...
+- **Complete** folder. The completed sample. To use this version, follow the instructions in the article [Create an ASP.NET Office Add-in that uses single sign-on](https://docs.microsoft.com/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet), but substitute "Complete" for "Begin" in those instructions and skip the sections **Code the client-side** and **Code the server-side**.
```
Comment (AlexJerabek) [inline Samples/auth/Office-Add-in-ASPNET-SSO/README.md:29]:
```suggestion
- **Complete** folder. The completed sample. To use this version, follow the instructions in the article [Create an ASP.NET Office Add-in that uses single sign-on](https://learn.microsoft.com/office/dev/add-ins/develop/create-sso-office-add-ins-aspnet), but substitute "Complete" for "Begin" in those instructions and skip the sections **Code the client-side** and **Code the server-side**.
```

File: Samples/auth/Office-Add-in-ASPNET-SSO/README.md [hunk 3]
```diff
@@ -23,39 +23,34 @@ extensions:
...
+- See how to use the Single Sign-on (SSO) API.
+- Access Microsoft Graph on behalf of an Office Add-in.
+- Build an add-in using ASP.NET Core, MSAL 4.x.x for .NET, and Office.js.
+- Use the MSAL.NET Library to implement the OAuth 2.0 authorization framework in an add-in.
+- Use the OneDrive REST APIs from Microsoft Graph.
 
 ## Applies to
 
--  Any platform and Office host combination that supports the [IdentityAPI 1.3 requirement set](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/identity-api-requirement-sets).
+- Any platform and Office host combination that supports the [IdentityAPI 1.3 requirement set](https://docs.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/identity-api-requirement-sets).
```
Comment (AlexJerabek) [inline Samples/auth/Office-Add-in-ASPNET-SSO/README.md:43]:
```suggestion
- Any platform and Office host combination that supports the [IdentityAPI 1.3 requirement set](https://learn.microsoft.com/en-us/office/dev/add-ins/reference/requirement-sets/identity-api-requirement-sets).
```

**Review by AlexJerabek — APPROVED** [review]
Approving with some link fix suggestions.

**Pull request merged**